README.rst

```diff
@@ -92,3 +92,9 @@ You want to tell us about it – best of all!
 Start at the `contributing guide <http://matplotlib.org/devdocs/devel/contributing.html>`_!
 
 Developer notes are now at `Developer Discussions <https://github.com/orgs/matplotlib/teams/developers/discussions>`_ (Note: For technical reasons, this is currently only accessible for matplotlib developers.)
+
+Citing Matplotlib
+==========
+Matplotlib is the brainchild of John Hunter (1968-2012), who, along with its many contributors, have put an immeasurable amount of time and effort into producing a piece of software utilized by thousands of scientists worldwide.
+If Matplotlib contributes to a project that leads to a scientific publication, please acknowledge this work by citing the project.
+`A ready-made citation entry <https://matplotlib.org/citing.html/>`_ is available.
```

Comment:
There's an extra backslash at the end of the link which means it's broken currently.